src/components/editor-content/editor-content.tsx

```diff
@@ -20,27 +21,42 @@ import { ErrorContext } from "components/error-handler/error-handler";
 import DeleteIcon from "@mui/icons-material/Delete";
 import vacationDaysProcess from "utils/vacation-data-utils";
 import { selectAuth } from "features/auth/auth-slice";
+import { TabContext, TabList, TabPanel } from "@mui/lab";
+import TestRangePicker from "components/generics/vacation-test-forms/myVacationComponent";
+import Holidays from "date-holidays";
+import vacationRequests from "components/generics/vacation-test-forms/testVacationMockData.json"
+import DateFilterPicker from "components/generics/date-range-picker/test-date-range-picker";
+import KeyboardArrowUpIcon from '@mui/icons-material/KeyboardArrowUp';
+import KeyboardArrowDownIcon from '@mui/icons-material/KeyboardArrowDown';
+import myVacationRequestsData from 'components/generics/vacation-test-forms/myVacationMockData.json';
+import renderVacationRequests from "components/generics/vacation-test-forms/myVacationRequests";
+import renderEmployeeVacationRequests from "components/generics/vacation-test-forms/employeeVacationRequests";
+import MyVacationsDatePicker from "components/generics/vacation-test-forms/myVacationsDatePicker";
 
 /**
- * Component properties
- */
+* Component properties
+*/
 interface Props {
 }
 
 /**
- * Application editor content component
- *
- * @param props component properties
- */
+* Application editor content component
+*
+* @param props component properties
+*/
 const EditorContent: React.FC<Props> = () => {
   const classes = useEditorContentStyles();
 
   const { person, personTotalTime } = useAppSelector(selectPerson);
   const { accessToken } = useAppSelector(selectAuth);
   const [ scope, setScope ] = React.useState<FilterScopes>(FilterScopes.WEEK);
   const [ dateFormat, setDateFormat ] = React.useState<string | undefined>("yyyy.MM.dd");
+  const [ dateVacationFormat, setDateVacationFormat ] = React.useState<string | undefined>("yyyy.MM.dd");
   const [ datePickerView, setDatePickerView ] = React.useState<CalendarPickerView>("day");
+  const [ datePickerViewVacation, setDatePickerViewVacation ] = React.useState<CalendarPickerView>("day");
```

Comment:
As mentioned earlier 
```suggestion
  const [ datePickerViewVacation, setDatePickerViewVacation ] = useState<CalendarPickerView>("day");
```